views/js/qtiXmlRenderer/renderers/Container.js

```diff
@@ -79,4 +81,27 @@ define(['tpl!taoQtiItem/qtiXmlRenderer/tpl/container'], function(tpl){
             return data;
         }
     };
+
+    /**
+     * Merges specified sibling html tags into one
+     * @param {string} html
+     * @returns {string}
+     */
+    function mergeSiblings(html)
+    {
+        if (typeof html !== 'string' || html.length <= 0) {
+            return html;
+        }
+        const tagsToMerge = [
+            'strong',
+            'em',
+            'sub',
+            'sup'
+        ];
+        tagsToMerge.forEach(function(tag) {
+            let regex = new RegExp("<\\/"+tag+">(\\s*)<"+tag+">", 'gi');
```

Comment:
```suggestion
            let regex = new RegExp("<\/"+tag+">(\s*)<"+tag+">", 'gi');
```
no need double `\\` in js

Reply:
sorry, it is ok
`If using the RegExp constructor with a string literal, remember that the backslash is an escape in string literals, so to use it in the regular expression, you need to escape it at the string literal level. /a\*b/ and new RegExp("a\\*b") create the same expression, which searches for "a" followed by a literal "*" followed by "b".`